app/views/backend/base/_state_bar.html.haml

```diff
@@ -10,7 +10,11 @@
       - classes << "active"
       %span{class: classes}= localized_name
     - elsif transition = transitions.detect{|t| t.to == state.name.to_s }
-      = link_to(localized_name, {action: transition.event, id: resource.id}, class: classes, method: :post, data: {confirm: :are_you_sure.tl}, title: transition.event.to_sym.ta)
```

Comment:
Hello Sandeep, this would technically work I guess but this is unsustainable, we cannot add special cases for each transition we want to disable. Could you take a look at the `state_machine` in the FixedAsset model instead ? You can find an example on how to use a condition on one of the transitions in the `state_machine` block of the model Parcel (line 122), rather than having to modify the html itself.